src/sentry/web/api.py

```diff
@@ -587,6 +587,53 @@ def process(self, request, project, key, auth, helper, data, attachments=None, *
                              key, remote_addr, helper, attachments)
 
 
+class EventAttachmentStoreView(StoreView):
+
+    def post(self, request, project, event_id, **kwargs):
+        if not features.has('organizations:event-attachments',
+                            project.organization, actor=request.user):
+            raise APIForbidden("Event attachments are not enabled for this organization.")
+
+        if len(request.FILES) == 0:
```

Comment:
I would just silently not do anything instead and return a 200 OK.

Reply:
@mitsuhiko I would vote for keeping it like this for now and only change if there's a customer use case to do so. It can help debug SDK development etc. 
Note that we do allow upload of empty files, which is definitely a valid case.

Reply:
@mitsuhiko: @jan-auer and I discussed this and I was already pushing to return not 2xx knowing you would rather have a 200.
At least an SDK with `Debug=true` could indicate what's going on instead of having a 200 with a failing request.
Moving from 400 to 200 is a non-breaking change so we can do it later if needed.